curvefs/src/client/fuse_client.cpp

```diff
@@ -180,6 +209,9 @@ CURVEFS_ERROR FuseClient::FuseOpOpen(fuse_req_t req, fuse_ino_t ino,
                            << ", inodeid = " << ino;
                 return CURVEFS_ERROR::INTERNAL;
             }
+            inode.set_length(0);
+            inode.set_ctime(nowTime);
+            inode.set_mtime(nowTime);
```

Comment:
should we set `access time` to current time?

Reply:
no need